source/java/src/main/java/com/uber/neuropod/LibraryLoader.java

```diff
@@ -0,0 +1,11 @@
+package com.uber.neuropod;
```

Comment:
I think you forgot to remove this file (there's already one under `source/neuropod/bindings/java`)